app/models/role.rb

```diff
@@ -26,6 +26,14 @@ def responsibilities
     details["body"]
   end
 
+  def organisations
+    links["organisations"]
```

Comment:
```suggestion
    links["ordered_parent_organisations"]
```

`organisations` is actually an older link type which is deprecated now, instead we should use `ordered_parent_organisations` because it makes sure the organisations will appear in the right order.

Reply:
Thanks, I'll update this 👍